### PR TITLE
fix: ui up to date during auction flow

### PR DESCRIPTION
**Pull request by creativogee**
## Description
Keep UI data up to date all time during auction/bid flow

### CODE REVIEW

File: src/features/partials/AssetDetails/SalesHistoryPanel.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ const SalesHistoryPanel = ({ data }: { data: SalesHistory[] }) => {
                 alt=''
                 className='rounded-full skeleton h-8 w-8'
               />
-              <span>{truncate(his.name, 6)}</span>
+              J<span>{truncate(his.name, 6)}</span>
```
Comment (jmulq) [inline src/features/partials/AssetDetails/SalesHistoryPanel.tsx]: Whats with the `J` here?

File: src/features/partials/modals/BuyBidError.tsx [hunk 2]
```diff
@@ -14,15 +14,15 @@ const BuyBidError = () => {
         </div>
         <h1 className='text-2xl my-4'>Error</h1>
         <p className='text-gray-500 text-center'>
-          There was an error submitting your bid. Please check your wallet
-          details and try again.
+          {error ??
+            ' There was an error submitting your bid. Please check your wallet details and try again.'}
```
Comment (jmulq) [inline src/features/partials/modals/BuyBidError.tsx:18]: Is space at front of string intentional?

File: src/features/partials/modals/SubmitBid.tsx [hunk 3]
```diff
@@ -63,8 +63,12 @@ const SubmitBid = () => {
       openModal('', <BidSubmitted />)
     }
 
-    if (isError) openModal('', <BuyBidError />)
-  }, [isApproved, isSigned, isSuccess, isError, openModal])
+    if (error) {
+      console.log('error', error)
```
Comment (jmulq) [inline src/features/partials/modals/SubmitBid.tsx:67]: Console.error instead? Do we need the log here?

**Review by jmulq — CHANGES_REQUESTED** [review]

File: src/features/partials/AssetDetails/SalesHistoryPanel.tsx (hunk 1 above)
Comment (creativogee) [reply src/features/partials/AssetDetails/SalesHistoryPanel.tsx]: saw that late. fixed

**Review by jmulq — APPROVED** [review]

File: src/features/partials/modals/SubmitBid.tsx (hunk 3 above)
Comment (creativogee) [reply src/features/partials/modals/SubmitBid.tsx]: Removing

**Pull request merged**